### PR TITLE
feat: python sdk doc (En)

### PR DESCRIPTION
**Pull request by ShiKaiWi**
Add the English docs for python sdk usage.

### CODE REVIEW

File: docs/src/en/sdk/python.md [hunk 1]
```diff
@@ -1 +1,136 @@
...
+
+- Python >= 3.7
+
+## Installation
+
+```bash
+pip install ceresdb-client
+```
+
+from ceresdb_client import Builder, PointBuilder, ValueBuilder, WriteRequest, SqlQueryRequest, Mode
```
Comment (MachaelLee) [inline docs/src/en/sdk/python.md:21]: remove this line? I can't get it..

Comment (ShiKaiWi) [reply docs/src/en/sdk/python.md:21]: Oops! It's a mistake, and will be removed.

File: docs/src/en/sdk/python.md [hunk 2]
```diff
@@ -1 +1,134 @@
...
+req = SqlQueryRequest(['demo'], create_table_sql)
+rpc_ctx = RpcContext()
+rpc_ctx.database = 'public'
+rpc_ctx.timeout_ms = 100
+event_loop = asyncio.get_event_loop()
+event_loop.run_until_complete(async_query(client, rpc_ctx, req))
+```
+
+`RpcContext` can be used to overwrite the default database and timeout defined in the initialization of the client.
+
```
Comment (MachaelLee) [inline docs/src/en/sdk/python.md:79]: Give an example of dropping table?

Comment (ShiKaiWi) [reply docs/src/en/sdk/python.md:79]: 👌

**Review by MachaelLee — APPROVED** [review]
LGTM

**Pull request merged**